readthedocs/projects/tasks/builds.py

```diff
@@ -777,7 +817,7 @@ def set_valid_clone(self):
         self.data.project.has_valid_clone = True
         self.data.version.project.has_valid_clone = True
 
-    def store_build_artifacts(self):
+    def store_build_artifacts(self, artifacts):
```

Comment:
Need to add this to the docstring.

Reply:
In the next iteration of this code, I removed it again because it was not strictly required: https://github.com/readthedocs/readthedocs.org/pull/9941/files#diff-4eae6664ca2124d124dbfe04a8000483c7a23d426c3f390721d6e80fe5891064L820